backend/actions/address_detail.js

```diff
@@ -12,47 +12,46 @@ app.use((req, res, next) => {
 });
 
 app.get('/address/:address', async (req, res) => {
-  let perPage = Number(req.query.perPage);
-  const page = Number(req.query.page);
+  // TODO: This API only supports returning 25 records for one request.
+  const lastSeenTxid = req.query.lastSeenTxid;
+  const address = req.params.address;
 
-  const regex = new RegExp(/^[0-9a-zA-Z]{26,35}$/);
-  const urlAddress = req.params.address;
-
-  if (!regex.test(urlAddress)) {
-    logger.error(`Regex Test didn't pass for URL - /address/${urlAddress}`);
+  if (!/^[0-9a-zA-Z]{26,35}$/.test(address)) {
```

Comment:
The address should be verified including checksum of Base58check encoding and version byte.

Reply:
Use tapyrus.address.fromBase58Check function (9f7301fa93b854bbf1966a272ced505382666df8)